tools/cravatp_score_and_annotate/cravatp_submit.xml

```diff
@@ -0,0 +1,306 @@
+<tool id="cravatp_submit" name="CRAVAT-P Submit, Intersect, Check, and Retrieve" version="1.0.0">
+    <description>| Submits, checks for, and retrieves data for cancer annotation. Can support proteogenomic proBED inputs for intersection with genomic VCF input.</description>
+  <command detect_errors="aggressive" interpreter="python">
+  <![CDATA[
+#if $proteo.proteoInput == 'yes':
+    #if $analysis.type == 'CHASM' or $analysis.type == 'CHASM+VEST':
+        cravatp_submit.py $input $GRCh $Variant $Gene $Noncoding $Error $analysis.type --classifier $analysis.classifier --proBED $proBED --intersectOnly $proteo.intersectedVariants --vcfOutput $vcf_output
+    #else: 
+        cravatp_submit.py $input $GRCh $Variant $Gene $Noncoding $Error $analysis.type --proBED $proBED --intersectOnly $proteo.intersectedVariants --vcfOutput $vcf_output 
+    #end if
+#else:
+    #if $analysis.type == 'CHASM' or $analysis.type == 'CHASM+VEST':
+        cravatp_submit.py $input $GRCh $Variant $Gene $Noncoding $Error $analysis.type --classifier $analysis.classifier 
+    #else: 
+        cravatp_submit.py $input $GRCh $Variant $Gene $Noncoding $Error $analysis.type
+    #end if
+#end if
+]]></command>
+  
+  <inputs>
+    <param format="tabular" name="input" type="data" label="Source file" help="Accepts transcriptomic or genomic inputs (e.g., tabular, VCF). Additional details can be found below."></param>
+    <conditional name="proteo">
+      <param format="tabular" name="proteoInput" type="select" display="radio" label="Intersect with proteogenomic input?" help="Source file (first input) must be in genomic input to enable intersection with this proteogenomic file.">
+        <option value="yes">Yes</option>
+        <option value="no" selected="true">No</option>
+      </param>
+      <when value="yes">
+        <param format="tabular" name="proBED" type="data" label="Peptides with Genomic Coordinates (ProBED Format)"></param>
```

Comment:
I think JJ added the proBED format already to Galaxy. So format should be probed https://github.com/galaxyproject/galaxy/blob/dev/config/datatypes_conf.xml.sample#L52

Reply:
The proteogenomics gateway does not have proBED yet. I'm thinking I will add this specification later.

Reply:
Can we update or patch the proteomic gateway?

Reply:
Yes. That would be best. I'm wrote an e-mail to @chambm about this with the link that you provided.

Reply:
I thought a main point of the proBAM and proBED formats was that they would be backward-compatible with existing BAM and BED processing tools. I know this is the case with proBAM at least. Tools that know nothing about proBAM are still able to process proBAM files properly. But I see there is a probam format in the new datatypes as well. I see proBED is a subclass of interval. I don't know much about proBED. Can BED-6 or BED-12 processing tools use proBED and get some benefit from the PSM information? If so it seems like having a separate type could be problematic.